diff --git a/bin/dot b/bin/dot
index 04eea8e..b4275f5 100755
--- a/bin/dot
+++ b/bin/dot
@@ -4,8 +4,8 @@
 set -euo pipefail
 
 ##### Start of Homebrew Installation Patch #####
-# export HOMEBREW_SLOTH=true
-# export SLOTH_PATH="HOMEBREW_PREFIX/opt/dot"
+export HOMEBREW_SLOTH=true
+export SLOTH_PATH="HOMEBREW_PREFIX/opt/dot"
 ##### End of Hombrew Installation Patch #####
 
 # In Linux we can do this with readlink -f but will fail in macOS and BSD OS
diff --git a/bin/up b/bin/up
index 711a9c0..b14de73 100755
--- a/bin/up
+++ b/bin/up
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
 ##### Start of Homebrew Installation Patch #####
-# export HOMEBREW_SLOTH=true
-# export SLOTH_PATH="HOMEBREW_PREFIX/opt/dot"
+export HOMEBREW_SLOTH=true
+export SLOTH_PATH="HOMEBREW_PREFIX/opt/dot"
 ##### End of Hombrew Installation Patch #####
 
 if [[ -z "${SLOTH_PATH:-}" ]]; then
diff --git a/scripts/core/install b/scripts/core/install
index 3b5c254..661a257 100755
--- a/scripts/core/install
+++ b/scripts/core/install
@@ -4,8 +4,8 @@
 set -euo pipefail
 
 ##### Start of Homebrew Installation Patch #####
-# export HOMEBREW_SLOTH=true
-# export SLOTH_PATH="HOMEBREW_PREFIX/opt/dot"
+export HOMEBREW_SLOTH=true
+export SLOTH_PATH="HOMEBREW_PREFIX/opt/dot"
 ##### End of Hombrew Installation Patch #####
 
 if [[ -z "${SLOTH_PATH:-}" ]]; then
diff --git a/scripts/core/loader b/scripts/core/loader
index 2d777ee..0f0cf34 100755
--- a/scripts/core/loader
+++ b/scripts/core/loader
@@ -4,8 +4,8 @@
 set -euo pipefail
 
 ##### Start of Homebrew Installation Patch #####
-# export HOMEBREW_SLOTH=true
-# export SLOTH_PATH="HOMEBREW_PREFIX/opt/dot"
+export HOMEBREW_SLOTH=true
+export SLOTH_PATH="HOMEBREW_PREFIX/opt/dot"
 ##### End of Hombrew Installation Patch #####
 
 [[ -z "${SLOTH_PATH:-${DOTLY_PATH:-}}" ]] && exit 1
diff --git a/scripts/core/short_pwd b/scripts/core/short_pwd
index e82d5cc..6e3d0df 100755
--- a/scripts/core/short_pwd
+++ b/scripts/core/short_pwd
@@ -3,8 +3,8 @@
 set -euo pipefail
 
 ##### Start of Homebrew Installation Patch #####
-# export HOMEBREW_SLOTH=true
-# export SLOTH_PATH="HOMEBREW_PREFIX/opt/dot"
+export HOMEBREW_SLOTH=true
+export SLOTH_PATH="HOMEBREW_PREFIX/opt/dot"
 ##### End of Hombrew Installation Patch #####
 
 . "${SLOTH_PATH:-${DOTLY_PATH:-}}/scripts/core/src/_main.sh"
diff --git a/shell/init-sloth.sh b/shell/init-sloth.sh
index c4ca67a..75c03ea 100644
--- a/shell/init-sloth.sh
+++ b/shell/init-sloth.sh
@@ -24,8 +24,8 @@ function recent_dirs() {
 }
 
 ##### Start of Homebrew Installation Patch #####
-# export HOMEBREW_SLOTH=true
-# export SLOTH_PATH="HOMEBREW_PREFIX/opt/dot"
+export HOMEBREW_SLOTH=true
+export SLOTH_PATH="HOMEBREW_PREFIX/opt/dot"
 ##### End of Hombrew Installation Patch #####
 
 # Advise no vars defines
